prism/src/main/java/org/apache/falcon/resource/AbstractExtensionManager.java

```diff
@@ -119,7 +117,7 @@ public int compare(ExtensionJobsBean o1, ExtensionJobsBean o2) {
             }
         };
 
-        Map<String, String> jobAndExtensionNames = new HashMap<>();
+        TreeMap<String, String> jobAndExtensionNames = new TreeMap<>();
```

Comment:
Won't this cause the jobs be returned in ascending order even when user says desc?